backend/app/views/spree/admin/shared/_head.html.erb

```diff
@@ -6,7 +6,15 @@
 <title><%= admin_page_title %></title>
 
 <%= favicon_link_tag 'favicon.ico' %>
-<%= stylesheet_link_tag 'spree/backend/all', media: 'all', data: {turbolinks_track: 'reload'} %>
+
+<% Spree::Config.admin_themes.each do |name, path| %>
+  <% media_queries = [] %>
+  <% media_queries << '(prefers-color-scheme: dark)' if Spree::Config.default_admin_dark_theme.to_s == name.to_s %>
+  <% media_queries << '(prefers-color-scheme: light)' if Spree::Config.default_admin_light_theme.to_s == name.to_s %>
```

Comment:
Same here, there's no dark theme yet if I got it correctly.

Reply:
Yes exactly, it's just the option to set different themes for when the system is in dark/light mode, similarly to how it works for other software (GitHub, Slack, etc).

Reply:
Got it. I think it's the right approach, but honestly, it still seems a bit out of context for this PR. Considering the end goal is to dismiss the old backend, I can't see much value in adding this feature now. 

It will take time to implement the full switch, so do you think it's essential to have it in the meantime? If you ask me, I personally don't, but at the same time, I don't think it will be the end of the world if we merge this enhancement so I won't block the PR for this.

Reply:
> do you think it's essential to have it in the meantime?

Yes, although "essential" is a stronger word than I would pick. Having been through many rewrites and migrations I know that holding on sensible and quality of life fixes doesn't pay off, since the turning point seems around the corner but usually proves to be just a mirage and the release date is shifted by another 3 months, and so on and so forth. 

This is also very portable to the new admin in which seems likely that we'll use [tailwind that lends itself very well to recompiling with custom palettes and themes](https://tailwindcss.com/docs/theme) that can be light and dark, allowing stores that are inclined to do so to match the admin to their brand.

Here's for example how GitHub allows you to pick a theme for each of dark and light modes:

<img width="1020" alt="image" src="https://github.com/solidusio/solidus/assets/1051/d89bcb68-ada7-47c1-bff8-e90abc2871c1">

Reply:
I'm also a bit concerned that we don't need it right now. Plus, it introduces a warning to end users, but I won't block it here either if you think it can help! In the end, it's an improvement...

Reply:
I think I'd prefer to add it. Otherwise, it's like having something half-done. We should probably add a new section about how to create your own theme here: https://guides.solidus.io/customization/customizing-the-backend